content/applications/finance/fiscal_localizations/united_arab_emirates.rst

```diff
@@ -0,0 +1,213 @@
+the **European Central Bank**. To do so, click on the drop-down menu of the :guilabel:`Service`
+field and select one of the services.
+
+.. _uae/payroll:
+
+Payroll
+=======
+
+The :guilabel:`U.A.E. - Payroll` module includes all the rules configured under the
+:guilabel:`U.A.E. Employee Payroll Structure` in the **Salary Rules** section as per the U.A.E.
+rules and regulations. These rules are linked to the corresponding accounts in the **Chart of
+Accounts**.
+
+.. image:: united_arab_emirates/uae-localization-salary-rules.png
+   :align: center
+   :alt: The U.A.E. Employee Payroll Structure.
+
+Salary rules
+------------
+
+To apply these rules to an employee's contract, go to :menuselection:`Payroll --> Contracts -->
+Contracts` and select the employee's contract. In the :guilabel:`Salary Structure Type` field,
+select :guilabel:`UAE Employee`.
+
+.. image:: united_arab_emirates/uae-localization-salary-structure.png
+   :align: center
+   :alt: Select the Salary Structure Type to apply to the contract.
+
+Under the :guilabel:`Salary Information` tab, you can find details such as the:
+
+- :guilabel:`Wage`;
+- :guilabel:`Housing Allowance`;
+- :guilabel:`Transportation Allowance`;
+- :guilabel:`Other Allowances`;
+- :guilabel:`Number of Days`: used to calculate :ref:`End of Service Provision
+  <ae-end-of-service-provision>`.
+
+.. note::
+   **Leave deduction** is calculated using a salary rule linked to the **Unpaid leave** time-off
+   type; however, any other deductions or reimbursements are made *manually* using other inputs.
+   Additionally, **overtime** is added *manually* through :menuselection:`Work Entries --> Work
+   Entries`, and **salary attachments** are generated from :menuselection:`Contracts --> Salary
+   Attachments`. :guilabel:`Create` an attachment and select the :guilabel:`Employee` and the
+   :guilabel:`Type (Attachment of Salary, Assignment of Salary, Child Support)`.
+
+.. tip::
+   If you wish to prevent a rule appearing on a paycheck, go to :menuselection:`Payroll -->
+   Configuration --> Structures`. Click on the **U.A.E. structure**, select the rule to hide, and
+   uncheck :guilabel:`Appears on Payslip`.
+
+.. _ae-end-of-service-provision:
+
+End of Service Provision
+------------------------
+
+The provision is defined as the total monthly allowance *divided* by 30 and then *multiplied* by the
+number of days set in the field :guilabel:`Number of days` at the bottom of a contract's form.
+
+The provision is then calculated via a salary rule associated with two accounts: the **End Of
+Service Indemnity (Expense account)** and the **End of Service Provision (Non-current Liabilities
+account)**. The latter is used to pay off the **End of service** amount by settling it with the
+**Payables account**.
+
+.. note::
+   The **End of service** amount is calculated based on the gross salary and the employee’s contract
```

Comment:
employee's contract's start and end dates
(or the start and end dates of the employee's contract)